### DISCUSSION TITLE
Should the KRE require apps to allow withdrawal?

### DISCUSSION THREAD

**Discussion by geoffwhittington** [kinfoundation/kin-rfcs#155] [Polls]
The KRE does not discriminate whether apps allow withdrawal of KIN from the app. A non-exhaustive list of potential side-effects due to lack of withdrawal is as follows:

* Successful apps are able to accumulate and control large amounts of KIN.
* Limits initial KIN integration work. This can be good when onboarding large apps.
* The pooled KIN is left unproductive to the broader ecosystem. It is unknown whether this trapped KIN is effectively "burned" or will be unleashed into broader circulation at some future point.
* Isolates the app from the other ecosystem apps, disallowing inter-app transfers and network effects that could otherwise emerge.
* Demotivates long-term KIN token holders who seek to realize the token's vision.

**Comment by Freedom-kin** [comment]
Unfortunately this is a logical side effect of a KRE based on AUB. The payout to the developers depends on the highest possible user balance. So they freeze the 1st 100,000 Kin or only let you withdraw once a month....anything to keep the balance from going lower. We can't blame the developers for this. They act within the limits of the KRE rules. It's the rules that need to change.

If we let go of the AUB, all those problems will resolve themselves, including the one you raise about allowing withdrawals.

**Comment by geoffwhittington** [comment]
I polled the app developers in Discord and as of this reply the result is 10-2 in favour of requiring withdrawal.

**Comment by perahodlr** [comment]
This is great. This should be part of the KIN ethos. If nothing else from the whitepaper is carried over, let this be the one that does. What is an ecosystem where you cannot bring your KIN from one app to another?

**Comment by Rachel1-2** [comment]
I personally think no withdrawal should mean no KRE.

**Comment by ghost** [comment]
It's a no brainer : apps should have withdrawal opens 24/7.  It should have been like this since the very beginning. And users should also have access to 100% of their KIN. No more gaming like "you can't withdraw the KIN that you earned within our app" just so the KIN gets locked to increase their own AUB (and get higher KRE reward).

KIN is a currency just like USD or CAD or whatever. When we log in our bank account, we have access to 100% of our money. Why would this be different with KIN? Apps only give those "non withdrawable" KIN for their own benefits (can't blame them because this behavior is allowed). But it's funny that some apps never did it (probably because of their own personal values which is remarkable) but others had absolutely no problem doing it since the beginning.

Make it fair for everyone : make it so apps need to open withdrawals 24/7 and for 100% of their user balance. It will be fair for all ecosystem apps and also fair for every KIN users. KIN will be stronger in a fair environment!

**Reply by Freedom-kin** [reply]
Totally agree.
It will significantly improve the user experience. It is inexplicable if users do not have access to money that is theirs.
